spec/models/event_spec.rb

```diff
@@ -41,4 +41,31 @@
       it { is_expected.to_not be_valid }
     end
   end
+
+  describe '.requiring_notification' do
+    subject(:events_requiring_notification) { Event.requiring_notification }
```

Comment:
No need to change this, is good to know that when you are referencing the class that you are testing in RSpec, you can use the `described_class` helper.

Makes it easier to change the name of the class if you ever had to